packages/playwright-core/src/server/chromium/videoRecorder.ts

```diff
@@ -137,10 +143,12 @@ export class VideoRecorder {
       for (let i = 0; i < repeatCount; ++i)
         this._frameQueue.push(this._lastFrameBuffer);
       this._lastWritePromise = this._lastWritePromise.then(() => this._sendFrames());
+      this._lastFrameTimestamp += repeatCount / fps;
```

Comment:
This shouldn't make any difference as the number of frames we feed into ffmpeg stays the same or am I missing something?